pkg/cmd/objects/update/object.go

```diff
@@ -1,78 +1,78 @@
-package updateObjects
+package update
 
 import (
 	"encoding/json"
 	"fmt"
 
 	"github.com/algolia/algoliasearch-client-go/v3/algolia/search"
 	"github.com/algolia/cli/pkg/utils"
+	"github.com/mitchellh/mapstructure"
 )
 
-type ObjectsToUpdate []map[string]interface{}
+// Object is a map[string]interface{} that can be unmarshalled from a JSON object
```

Comment:
Good habit to add comments to describe `Object` type for such a generic word, I'll try to do the same in the future 🔥